python/lsst/daf/butler/queries/tree/__init__.py

```diff
@@ -35,6 +35,5 @@
 from ._query_tree import *
 
 LogicalNot.model_rebuild()
-del LogicalNot
```

Comment:
This `del` was crashing sphinx.  This is an internal module that isn't re-exported anywhere or referenced from the documentation so it doesn't super matter if there's an extra symbol in here.

Reply:
I think it was being deleted so as not to be lifted into the parent on important. Another way to do it might be:

```python
from ._predicated import LogicalNot as _LogicalNot
...
_LogicalNot.model_rebuild()
```
which should have the same effect of not affecting the exports.